p2p/src/node.rs

```diff
@@ -250,6 +264,17 @@ where
                         }
                     }
                 }
+                provide_records = self.publisher.next() => {
+                    if let Some(kad) = self.swarm.behaviour_mut().kad.as_mut() {
+                        if let Some(provide_records) =  provide_records {
```

Comment:
```suggestion
                        if let Some(provide_records) = provide_records {
```
For some reason, my eyes are very good at catching extra/missing whitespaces 😂

Reply:
Hmm I would expect `cargo fmt` to catch this. I'll take a look.

Reply:
cargo fmt can't always format code inside of a macro and this code block lives inside a `select!` macro invocation. This is my I generally like to put as little logic inside the macro as possible and just call out to functions in the bodies of the branches. 

Fixed